src/demos/storage-access-api/personalization.ejs

```diff
@@ -9,90 +9,122 @@
     containerClass,
     hasStorageAccess;
 
+  // Fetch and apply the user's preferred theme from a remote URL
   async function fetchAndApplyTheme() {
     try {
+      // Send a GET request to the remote URL
       const response = await fetch(`${baseURL}/get-personalization`, {
         method: 'GET',
+        // Include the user's credentials in the request
         credentials: 'include',
     });
-
+      // Check if the response was successful
       if (!response.ok) {
         throw new Error('Network response was not ok');
+        // If the response was not successful, throw an error
         errorMessages.textContent = `Network response was not ok ${response.status} - ${response.statusText}`;
```

Comment:
This line needs to be changed, after the error is thrown, the subsequent lines are not executed/interpreted.